src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -277,56 +300,55 @@ public CommunityOrPrisonOffenderManager allocatePrisonOffenderManagerByNomsNumbe
     }
 
     public static class InvalidAllocatePOMRequestException extends BadRequestException {
-        InvalidAllocatePOMRequestException(CreatePrisonOffenderManager createPrisonOffenderManager, String message) {
+        InvalidAllocatePOMRequestException(final CreatePrisonOffenderManager createPrisonOffenderManager, final String message) {
             super(message);
             log.warn("Bad request: " + createPrisonOffenderManager);
         }
     }
 
     @RequestMapping(value = "/offenders/crn/{crn}", method = RequestMethod.GET)
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "The offender summary", response = OffenderDetailSummary.class),
+            @ApiResponse(code = 200, message = "The offender summary"),
```

Comment:
so are you keeping the 200 annotation when someone has added a message?

Reply:
agreed, the default is `OK`, so if they've added something else then keeping it